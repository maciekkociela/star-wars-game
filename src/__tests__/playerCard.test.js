import React from "react";
import PlayerCard from "../components/playerCard";
import renderer from "react-test-renderer";

it("renders correctly", () => {
    const tree = renderer
        .create(<PlayerCard player={1} />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
