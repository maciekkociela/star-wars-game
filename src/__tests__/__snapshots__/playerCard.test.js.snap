// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiPaper-root MuiPaper-elevation1 MuiCard-root MuiPaper-rounded"
  style={
    Object {
      "margin": 10,
      "width": 300,
    }
  }
>
  <div
    className="MuiCardContent-root"
  >
    <h5
      className="MuiTypography-root MuiTypography-h5"
    >
      Player 
      1
    </h5>
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary MuiTypography-gutterBottom"
    >
      Points: 
      0
    </p>
    <h6
      className="MuiTypography-root MuiTypography-h6"
    >
      1
    </h6>
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-colorTextSecondary"
    >
      
       
      
    </p>
  </div>
</div>
`;
